docs/dev/plugins/other/paths.rst

```diff
@@ -0,0 +1,58 @@
+Handling Paths
+==============
+
+Historically, this chapter recommended the utilities ``syspath()``,
+``normpath()``, ``bytestring_path()``, and ``displayable_path()`` for handling
+file paths in Beets. These ensured consistent behavior across Linux, macOS, and
+Windows before Python’s ``pathlib`` offered a unified and reliable API.
+
+- ``syspath()`` worked around Windows Unicode and long-path issues by converting
+  to a system-safe string (adding the ``\\?\`` prefix where needed). Modern
+  Python (≥3.6) handles this automatically through its wide-character APIs.
+- ``normpath()`` normalized slashes and removed ``./`` or ``..`` parts but did
+  not expand ``~``. It was used mainly for paths from user input or config
+  files.
+- ``bytestring_path()`` converted paths to ``bytes`` for storage in the
+  database. Paths in the database are still stored as bytes today, though there
+  are plans to eventually store ``pathlib.Path`` objects directly.
+- ``displayable_path()`` converted byte paths to Unicode for display or logging.
+
+These utilities remain safe to use when maintaining older code, but new code and
```

Comment:
Let's move the how to do it right now to the top. I feel like the flow of this section should be permuted. Maybe something like:
- How we want dev to handle paths right now
- Historic methods you might still find in the codebase
- Examples